packages/react-vapor/src/components/select/MultiSelectConnected.tsx

```diff
@@ -21,6 +21,7 @@ import {ISelectButtonProps, ISelectOwnProps, ISelectProps, SelectConnected} from
 import {SelectSelector} from './SelectSelector';
 
 export interface IMultiSelectOwnProps extends ISelectProps, IDropTargetProps {
+    toggleClasses?: string;

```

Comment:
Can we move this to `ISelectProps` instead? We're using it in both SingleSelect & MultiSelect